webapp/Connector/src/app/view/Study.js

```diff
@@ -45,9 +45,13 @@ Ext.define('Connector.app.view.Study', {
         resizable: false,
         dataIndex: 'assays_added_count',
```

Comment:
I've noticed an edge case with sorting on Data Added column. This is not a new issue, but with the change of filter from "# of Assay Added" to "Data Available", the issue has been made more obvious. The problem is, the sort is operated on 'assays_added_count', but the column displays other counts as well, such as 'publication count'. I think ideally, the sort should sort on "# of assay added", and if equal (or both null), then sort on other data types, such as sum of 'non-integrated added' and 'Publication data'. 

If we care to change the sorting behavior, a workaround I can think of is to add a new 'study_data_added_sort_ind' column, which is calculated as (assays_added_count*100 + non_integrated_count + publication_count).

Production site, sorted by 'Data Added":
![Screen Shot 2021-12-07 at 11 56 28 AM](https://user-images.githubusercontent.com/5168106/145098314-4765da2c-d704-4b49-ba84-2603a26ea255.png)


Local:
<img width="930" alt="Screen Shot 2021-12-07 at 11 56 40 AM" src="https://user-images.githubusercontent.com/5168106/145098453-36a8fcb8-20f7-449c-99f2-b146e9b110b6.png">

Reply:
Thanks Xing, this is a good point. I'll plan to bring this up with the team during our next CDS meeting.